### ISSUE TITLE
Input groups: Bugs

### ISSUE THREAD

**Issue by KateMacdonald** [WestpacGEL/GEL-next#668]
- [x] Simple way to change the colour of icons

- [x] Ability to make Icon input groups fixed width, when icon is 'after' it doesn't move with the field

- [x] The labels for the add-ons that go with the input group components should increase in size with the input field size. It seems to be OK for the Text add-ons but the **Select** and **Button** labels don't' change.
**Internal labels and input text sizes should be:**
Small: 14px
Med: 16px
Large: 16px
XLarge: 18px

![image](https://github.com/WestpacGEL/GEL-next/assets/6667337/9726b0a0-f8f7-4871-a3c0-43b5c6d71ec4)

**Comment by samithaf** [comment]
@KateMacdonald isn't it already simple to change the colours of the icon? See the following code snippet.

Simple way to change the colour of icons

```
after={{
          inset: true,
          element: <Button onClick={clearInput} look="link" iconAfter={ClearIcon} iconColor="success" />,
        }}
```

<img width="1481" alt="Screenshot 2024-01-22 at 9 24 45 pm" src="https://github.com/WestpacGEL/GEL-next/assets/1860417/2d900eef-7249-494d-9cb9-6b5f492ecaf1">

**Comment by jaortiz** [comment]
Fixed in #693
